contributing/development/compiling/compiling_for_windows.rst

```diff
@@ -214,6 +217,17 @@ Optionally, you can compile with the following for additional features:
 .. note:: If you use a preview version of the Agility SDK, remember to enable
```

Comment:
Should 'preview' be 'previous'?
"If you use a previous version..."

Reply:
Nope. If you check https://devblogs.microsoft.com/directx/directx12agility/, you'll see that they release _preview_ versions, which are like betas of upcoming releases. This sentence is about them.